lib/command.coffee

```diff
@@ -102,7 +102,7 @@ runner = ->
     .map((x)->resolve(x))
 
   port = opts.get('port') || 35729
-  host = opts.get('host') || 'localhost'
+  host = opts.get('bind') || 'localhost'
```

Comment:
@gorankarlic So we don't get confused in the future, can you please change this to `bindAddress` or `bindHost` 
instead of `host`throughout?

Reply:
@napcs Done.